dev/io.openliberty.checkpoint_fat_session_cache_infinispan_container/bnd.bnd

```diff
@@ -29,8 +29,7 @@ tested.features: \
   servlet-6.0,\
   componenttest-2.0,\
   cdi-3.0,\
-  cdi-4.0,\
-  checkpoint
```

Comment:
I think this one is passing regularly because it still uses an older version of infinispan.  I suggest we only take `checkpoint` out of the `dev/io.openliberty.checkpoint_fat_session_cache_infinispan/bnd.bnd` file.

Reply:
Done. Thanks!